<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Schedule_Edit_Screen</class>
 <widget class="QWidget" name="HNS_Schedule_Edit_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HNS_Date_Button" name="PB_Sched_EndDate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>210</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>SELECT
DATE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>720</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_NeverEnd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>NEVER END</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_StartNow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>START NOW</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Back">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>720</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Title">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>500</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141;border: 3px solid #414141;border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>SCHEDULE TITLE</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Sched_Preview" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>50</y>
     <width>291</width>
     <height>164</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Time_Button" name="PB_Sched_EndTime">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>92</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>SELECT
TIME</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>START SCHEDULE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>END SCHEDULE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_NewSched">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE NEW SCHEDULE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_EditMessage">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>220</y>
     <width>291</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>EDIT MESSAGE BOARD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Tuesday">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>TUESDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Saturday">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>SATURDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Friday">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>FRIDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="GV_DownArrow2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>42</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background: transparent
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Sunday">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>SUNDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="GV_DownArrow1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>42</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Monday">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>MONDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Thursday">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>THURSDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="HNS_Date_Button" name="PB_Sched_SelectStartDate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>210</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>SELECT
DATE</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="HNS_Time_Button" name="PB_Sched_StartTime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>92</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	border-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>SELECT
TIME</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_Wednesday">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>110</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#E8E8E8;
	color:#414141;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#FFAC27;
	color:#000000;
}</string>
   </property>
   <property name="text">
    <string>WEDNESDAY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Sched_Error">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>SCHEDULE INVALID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="HNS_Clock_Widget" name="LBL_Clock" native="true">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="HNS_Radar_Info_Widget" name="PGE_Radar" native="true">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>300</y>
     <width>280</width>
     <height>157</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_EditRadar">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>470</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>EDIT
RADAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_ClearRadar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>470</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CLEAR
RADAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_ClearStrobes">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>550</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CLEAR
STROBE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Sched_EditStrobes">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>550</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>EDIT
STROBE</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Clock_Widget</class>
   <extends>QWidget</extends>
   <header>hns_clock_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Time_Button</class>
   <extends>QPushButton</extends>
   <header>hns_time_button.h</header>
  </customwidget>
  <customwidget>
   <class>HNS_Date_Button</class>
   <extends>QPushButton</extends>
   <header>hns_date_button.h</header>
  </customwidget>
  <customwidget>
   <class>HNS_Radar_Info_Widget</class>
   <extends>QWidget</extends>
   <header>hns_radar_info_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
